apps/.gitignore

```diff
@@ -3,4 +3,9 @@
 **/dist/**/*.js*
 **/target/*
 **/build/*
-docs
\ No newline at end of file
+docs
```

Comment:
changes to the `docs` dir should not be ignored for sure

Reply:
removed it from .gitignore